packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -69,15 +70,67 @@ const confirmDialogsPropTypes = {
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+  selectedEntries,
+  slug,
+}) => {
   const { formatMessage } = useIntl();
+  const [numberOfDraftRelations, setNumberOfDraftRelations] = React.useState(null);
+  const { get } = useFetchClient();
+
+  const onDraftRelationCheck = React.useCallback(async () => {
+    try {
+      const endPoint = `/content-manager/collection-types/${slug}/actions/countManyEntriesDraftRelations?${stringify(
+        { ids: selectedEntries },
+        { encode: false }
+      )}`;
+
+      const draftRelations = await get(endPoint);
+      const draftEntries = await draftRelations.data.data;
+      setNumberOfDraftRelations(draftEntries);
+
+      return draftEntries;
+    } catch (err) {
+      console.error(err.message);
+
+      return Promise.reject(err);
```

Comment:
I don't understand why you're using Promise Reject? I think this will throw the entire component

Reply:
I forgot to remove it